--- /usr/share/pear/HTML/QuickForm/date.php~	2010-09-06 18:13:32.000000000 +0300
+++ /usr/share/pear/HTML/QuickForm/date.php	2011-01-07 14:29:28.506057805 +0200
@@ -55,8 +55,8 @@
     var $_options = array(
         'language'         => 'en',
         'format'           => 'dMY',
-        'minYear'          => 2001,
-        'maxYear'          => 2010,
+        'minYear'          => 2005,
+        'maxYear'          => 2020,
         'addEmptyOption'   => false,
         'emptyOptionValue' => '',
         'emptyOptionText'  => '&nbsp;',
